hyperjumptech/monika
# Feature: limit database

# Monika Pull Request (PR)  
fix #414 

## What feature/issue does this PR add  
Need to limit database size

## How did you implement / how did you fix it  
Add cron job for running tasks to delete data from db

## How to test  
1. set database limit to your configuration
```
db_limit:
  max_db_size: 1000000000
  deleted_data: 1
  cron_schedule: '*/1 * * * *'
```
2. run the application
3. check that the cron job should run and delete 1 data every minute
